exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
+var greetings = "Hello, my name is ";
```

Comment:
Hi Advocate, I see you are still using var to declare variables, as it was mentioned last week, that var is no longer being used much this days and we were advised to use "let" and "const"

Reply:
Network destructed me. Thank you, I will change the code